feat/EntityRestaurantMenu

AleCam04 opened the pull request.

Commit:
feat/EntityRestaurantMenu

dudinhab commented on src/main/java/com/develhope/Java27_progetto3_team2/entity/RestaurantMenu.java:
```diff
@@ -0,0 +1,21 @@
+import lombok.Setter;
+
+@Getter
+@Entity
```
Entity name is missing

DonatoAletta replied:
Also missing @AllArgsConstructor, @NoArgsConstructor

dudinhab commented on src/main/java/com/develhope/Java27_progetto3_team2/model/RestaurantMenuDTO.java:
```diff
@@ -0,0 +1,6 @@
+package com.develhope.Java27_progetto3_team2.model;
+
```
Methods appropriate Getters and Setters are missing

DonatoAletta replied:
Also missing @AllArgsConstructor, @NoArgsConstructor

DonatoAletta commented on src/main/java/com/develhope/Java27_progetto3_team2/entity/RestaurantMenu.java:
```diff
@@ -0,0 +1,21 @@
+@Getter
+@Entity
+public class RestaurantMenu {
+   @Id
```
Missing @GeneratedValue(strategy = GenerationType.IDENTITY)

dudinhab commented on src/main/java/com/develhope/Java27_progetto3_team2/entity/RestaurantMenu.java:
```diff
@@ -0,0 +1,21 @@
+   private int id;
+   @Setter
+   @Column(name = "menu")
+   private String menu;
```
private List<menuItem> menuItem

Commit:
fix/RestaurantMenu class: remove String menu, added List menuItemList and constructors, modify id and restaurantId.
fix/RestaurantMenuDTO class: remove String menu, added restaurantId, getter, constructors and List menuItemList, modify id

Fabio013 reviewed (approved):
Pr ok
